docs-md/tutorials/basics-docker.md

```diff
@@ -252,70 +294,153 @@ Optional: Show all streams and tables.
-PAGEVIEWS_ORIGINAL       | pageviews                | DELIMITED
+ Stream Name         | Kafka Topic                 | Format    
+---------------------------------------------------------------
+ KSQL_PROCESSING_LOG | default_ksql_processing_log | JSON      
+ PAGEVIEWS_ORIGINAL  | pageviews                   | DELIMITED 
+---------------------------------------------------------------
 
 ksql> SHOW TABLES;
 
-Table Name        | Kafka Topic       | Format    | Windowed
---------------------------------------------------------------
-USERS_ORIGINAL    | users             | JSON      | false
+ Table Name     | Kafka Topic | Format | Windowed 
+--------------------------------------------------
+ USERS_ORIGINAL | users       | AVRO   | false    
+--------------------------------------------------
 ```
 
+!!! tip
+    Notice the `KSQL_PROCESSING_LOG` stream listed in the SHOW STREAMS
+    output? KsqlDB will append messages detailing any issues it
+    encountered while processing your data. If things are not working
+    as you expect it can be worth checking the contents of this stream
+    to see if ksqlDB is encountering data errors.
+
+Viewing your data
+-----------------
+
+1. Use `SELECT` to create a query that returns data from a TABLE. This
+   query includes the `LIMIT` keyword to limit the number of rows
+   returned in the query result, and the `EMIT CHANGES` keywords to
+   indicate we wish to stream results back. This is known as a
+   [pull query](../concepts/queries/pull.md). See the
+   [queries](../concepts/queries/index.md) for an explanation of the
+   different query types. Note that exact data output may vary because
+   of the randomness of the data generation.
+   ```sql
+   SELECT * from users_original emit changes limit 5;
+   ```
+
+   Your output should resemble:
+
+   ```
+   +--------------------+--------------+--------------+---------+----------+-------------+
+   |ROWTIME             |ROWKEY        |REGISTERTIME  |GENDER   |REGIONID  |USERID       |
+   +--------------------+--------------+--------------+---------+----------+-------------+
+   |1581077558655       |User_9        |1513529638461 |OTHER    |Region_1  |User_9       |
+   |1581077561454       |User_7        |1489408314958 |OTHER    |Region_2  |User_7       |
+   |1581077561654       |User_3        |1511291005264 |MALE     |Region_2  |User_3       |
+   |1581077561857       |User_4        |1496797956753 |OTHER    |Region_1  |User_4       |
+   |1581077562858       |User_8        |1489169082491 |FEMALE   |Region_8  |User_8       |
+   Limit Reached
+   Query terminated
+   ```
+   !!! note
+         Push queries on tables will output the full history of the table that is stored
+         in the Kafka changelog topic, i.e. it will output historic data, followed by the
+         stream of updates to the table. It is therefore likely that rows with matching
+         `ROWKEY` are output as existing rows in the table are updated.
```

Comment:
I'm not sure when, but the default behaviour of push queries on tables has changed: they now output all data without the need to set `auto.offset.reset` session property.

There may be other docs that need updating to reflect this @JimGalasyn